app/components/code-sample/model.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const Prism = require('prismjs');
+require("prismjs/components/prism-yaml");
+
+module.exports.save = (uri, data) => {
+    //added manual spaces, klin codemirror don't recognize tab spaces
```

Comment:
Should probable add a whitespace after `//` or the linter will yell
```suggestion
    // Adds manual spaces, klin codemirror doesn't recognizes tab spaces
```